tests-e2e/features/assets/digital-twin-detail.feature

```diff
@@ -0,0 +1,79 @@
+Feature: Digital Twin Detail
+
+  Background:
+    Given I log in to evan.network using vue
+      And I click on "My Assets" in main menu
+      And I click on "Digital Twins" in sub menu
+      And I search in assets for "Test Car Twin"
+      And I click on an element with text including "Test Car Twin"
+      And I wait for 3 seconds
+      And I wait 20 seconds until loading was finished
+
+  Scenario: Open Digital Twin
+    Then I want to see a element with class "twin-name"
```

Comment:
we should refactor this to "I want to see an element with class..." to have correct english grammar here.